presto-native-execution/presto_cpp/main/common/Configs.h

```diff
@@ -90,6 +90,9 @@ class SystemConfig : public ConfigBase {
   static constexpr std::string_view kAsyncCacheSsdPath{"async-cache-ssd-path"};
   static constexpr std::string_view kEnableSerializedPageChecksum{
       "enable-serialized-page-checksum"};
+  static constexpr std::string_view kUseMmapArena{"use_mmap_arena"};
+  static constexpr std::string_view kMmapArenaCapacityRatio{
+      "mmap_arena_capacity_ratio"};
```

Comment:
I have impression (unconfirmed) that all config properties are using dashes ("-"), while session properties are using underscores ("_"). We should probably follow the trend.